### DISCUSSION TITLE
Startup Feature

### DISCUSSION THREAD

**Discussion by elozturk** [aviolaris/fritzbox-tray#2] [Q&A] [answered]
Hello there,

Many thanks for this tray app!! It is such a relief not having to open the browser and log in every time I need to find or change my IP. However I'm wondering if you can add an automatic startup feature. In Windows I just put it in the startup folder but I'm struggling to find an easy solution for my Ubuntu laptop.

Best regards,

Elif

**Comment by aviolaris** [comment, marked as answer]
Hello.

Thank you for your feedback! It is in my roadmap to include the requested feature in a future release of the app. However, in the meantime, you can easily add the app to your Ubuntu startup by executing the following command in your terminal:

```bash
PYTHON_PATH=$(command -v python || command -v python3) && echo -e "[Desktop Entry]\nType=Application\nName=FritzBox Tray\nExec=$($PYTHON_PATH -c "import site; print(site.USER_BASE)")/bin/fritzbox-tray\n" | tee ~/.config/autostart/fritzbox-tray.desktop
```
If you want to remove the app from your startup list, you can do so by running the command below:

```bash
rm ~/.config/autostart/fritzbox-tray.desktop
```

Regards!

**Reply by elozturk** [reply]
It looks like a command from a spaceship control panel but it worked fine :smile:
